Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -1533,24 +1533,18 @@ PhysicalParticleContainer::AddPlasmaFlux (PlasmaInjector const& plasma_injector,
         [=] AMREX_GPU_DEVICE (int i, int j, int k, amrex::RandomEngine const& engine) noexcept
         {
             const IntVect iv = IntVect(AMREX_D_DECL(i, j, k));
+            amrex::ignore_unused(j,k);
             const auto index = overlap_box.index(iv);
 
             Real scale_fac = compute_scale_fac_area(dx, num_ppc_real, flux_normal_axis);
 
             auto lo = getCellCoords(overlap_corner, dx, {0._rt, 0._rt, 0._rt}, iv);
             auto hi = getCellCoords(overlap_corner, dx, {1._rt, 1._rt, 1._rt}, iv);
 
-            if (flux_pos->overlapsWith(lo, hi))
```

Comment:
This `if` condition is not key here, since the `pcounts[index]` is `0` anyway for cells that do not overlap.